jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1049,7 +1049,7 @@ codet java_bytecode_convert_methodt::convert_instructions(
        i_it->statement=="monitorexit")))
     {
       const std::vector<unsigned int> handler =
-        try_catch_handler(i_it->address, method.exception_table);
+        try_catch_handler(i_it->address, method.handled_exception_table);
```

Comment:
I'm not sure about this one. How many exception tables are there?

Reply:
There are two. There this one which deals with exceptions that can be caught. And the one that I am parsing in this PR which are exceptions that a method throws. Strictly speaking, in the bytecode, this is called exception_table and mine is also called exception_table. I thought this just made it a bit clearer what is what.